packages/superset-ui-demo/storybook/shared/components/ResizableChartDemo.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, ReactNode } from 'react';
+import ResizablePanel from './ResizablePanel';
+
+interface Size {
+  width: number;
+  height: number;
+}
+
+export default function ResizableChartDemo({ children }: { children: (size: Size) => ReactNode }) {
+  // size are all inner size
+  const initialSize = { width: 500, height: 300 };
+  const [size, setSize] = useState(initialSize);
+  return (
+    <div className="superset-body">
```

Comment:
Is this className needed?

Reply:
Yes, it's for having a light gray background in main body (as `panel` is white).

Reply:
Could we do this in `js`/`css-in-js` to move away from declaring this in `storybook.css`?